README.md

```diff
@@ -235,3 +235,40 @@ validate that the pre-scripted responses matches the real responses.
 ```
 ./.ci/verify_py_test_data.sh
 ```
+
+### Running in a Docker Container
+
+The autograder can also be run from a [docker](https://www.docker.com/) container either
+by building the image from the source code with Dockerfile or pulling it from Docker Hub.
+
+The autograder container requires two mounts. The first is `/var/run/docker.sock`,
+the socket that the Docker daemon listens on. The second is the autograder's temporary directory `/tmp/autograder-temp/`.
+Both mounts rely on a POSIX system.
+
+To build the image form source code navigate to autograder-server/ directory and run:
+```
+docker build -f docker/Dockerfile -t autograder .
+```
+
+To run the container run:
+```
+docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock -v /tmp/autograder-temp/:/tmp/autograder-temp autograder <command>
+```
+
+The `<command>` can be any command form the `cmd` folder. For example it can be `version`:
+```
+docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock -v /tmp/autograder-temp/:/tmp/autograder-temp autograder version
+```
+
+If you want to run the server you need to add `-p` flag to the the command, as shown below:
```

Comment:
This is not entirely factual, you can run the server without passing the port.
Just note that it could be useful for the server.